### DISCUSSION TITLE
weird error

### DISCUSSION THREAD

**Discussion by 11bunno11** [vercel/community#7112] [Help]
### Summary

I get a 501 error whenever trying to add developer commands

### Steps to Reproduce

Whenever i try to add a dev command this happens
```
Unexpected response: text/html <!DOCTYPE html public "-//w3c//DTD HTML4.01 Transitional//EN">
 <html lang="ja">
 <head>
 <meta http-equiv="content-type" content="text/html;charset=UTF-8">
 <title>501 Not Implemented</title>
 </head>
 <body>
 <H2>501 Not Implemented</H2>
 </body>
 </html>
```

**Comment by leerob** [comment]
Hey there, could you include a minimal reproducible example please?

https://vercel.com/guides/creating-a-minimal-reproducible-example

**Reply by 11bunno11** [reply]
its more in the website itself, not the program
